pandas/tests/reshape/concat/test_concat.py

```diff
@@ -755,3 +756,49 @@ def test_concat_retain_attrs(data):
     df2.attrs = {1: 1}
     df = concat([df1, df2])
     assert df.attrs[1] == 1
+
+
+@td.skip_array_manager_invalid_test
```

Comment:
why is this invalid for AM?

Reply:
Because the AM has different behaviour, see last sentence in https://github.com/pandas-dev/pandas/issues/40893 (AFAIR a consequence of using `concat_compat`, which is basically the Series behaviour, which is also known to be inconsistent, xref https://github.com/pandas-dev/pandas/issues/39122).  
The revert of the original PR also introduced some other related test changes, see eg https://github.com/pandas-dev/pandas/pull/47372/files#diff-50f6426546495aad672032deb56b5b222b35697637f5a7f0353ec8ff33bd8ca5R187. And see also the last part of the comment at https://github.com/pandas-dev/pandas/pull/43507#issuecomment-924342478